qmp/socket.go

```diff
@@ -68,6 +68,26 @@ func NewSocketMonitor(network, addr string, timeout time.Duration) (*SocketMonit
 	return mon, nil
 }
 
+// NewListeningSocketMonitor listens for a monitor socket connection from Qemu.
+// An error is returned if the connection fails to accept.
```

Comment:
Can you expand this comment a bit - the text from your PR would be helpful here, so that users understand they can either connect to qemu or instruct qemu to connect to this process, depending on the parameters passed to qemu.

Reply:
No problem! I've updated the docs here with a few examples. Let me know if I'm missing anything